src/sentry/api/serializers/models/event.py

```diff
@@ -99,6 +99,11 @@ def serialize(self, obj, attrs, user):
             except TypeError:
                 received = None
 
+        event_type = obj.data.get('type', 'default')
+        metadata = obj.data.get('metadata') or {
+            'title': obj.message_short,
+        }
```

Comment:
OK, this was confusing at first, just going through `DefaultEvent` would make this clearer if we're not worried about inferring the values for past events like we're doing on the way in.